### PR TITLE
Fixed oldfrivia and allplayer page

### PR DESCRIPTION
**Pull request by Musealo**
Fixed both pages, added readme and changed some styling

### CODE REVIEW

File: pages/api/rooms/[room_id]/frivia/index.js [hunk 1]
```diff
@@ -32,11 +32,14 @@ export default async function handler(request, response) {
...
-              console.log(answer.userId._id.valueOf(), session.user.id);
               return answer.userId._id.valueOf() === session.user.id;
             });
 
-            if (request.query.oldFrivias && request.query.oldFrivias === true) {
+            if (
+              request.query.oldFrivias &&
+              request.query.oldFrivias === 'true'
+            ) {
+              console.log(request.query.oldFrivias);
```
Comment (Ju-Schn) [inline pages/api/rooms/[room_id]/frivia/index.js:42]: Do you still need this? ;)

File: pages/room/[room_id]/oldfrivia.js [hunk 2]
```diff
@@ -18,10 +18,7 @@ function OldFrivia() {
         `/api/rooms/${room_id}/frivia?oldFrivias=true`
       );
       let friviasData = await response.json();
-
-      friviasData = friviasData.filter(
-        notAnswered => notAnswered.userAnswered === true
-      );
+      console.log(friviasData);
```
Comment (Ju-Schn) [inline pages/room/[room_id]/oldfrivia.js:21]: 👀

**Review by Ju-Schn — APPROVED** [review]
Very nice work! 😊

File: pages/api/rooms/[room_id]/frivia/index.js (hunk 1 above)
Comment (Musealo) [reply pages/api/rooms/[room_id]/frivia/index.js:42]: Removed it, thanks!

File: pages/room/[room_id]/oldfrivia.js (hunk 2 above)
Comment (Musealo) [reply pages/room/[room_id]/oldfrivia.js:21]: Removed it, thanks!

**Pull request merged**